tests/lib/signinCodes.js

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern!tdd',
+  'intern/chai!assert',
+  'tests/addons/environment'
+], function (tdd, assert, Environment) {
+
+  // These tests are intended to run against a mock auth-server. To test
```

Comment:
Seems like only the `consumeSigninCode` will fail when using a real auth-server. Would it make sense to put this check in that test case only?